spec/factories/journal_entries.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :journal_entry do
+    journal_id {1}
```

Comment:
Writing only this should work, it is supposed to automatically create a journal when you create a journal_entry
```suggestion
    journal
```